app.js

```diff
@@ -1,10 +1,25 @@
 const express = require("express");
 const app = express();
-const { getTopics, getApi } = require("./controllers/topic.controller");
+const {
+  getTopics,
+  getApi,
+  getArticleId,
+} = require("./controllers/topic.controller");
 
-app.use(express.json());
+// app.use(express.json());
 
 app.get("/api/topics", getTopics);
 app.get("/api/", getApi);
+app.get("/api/articles/:article_id", getArticleId);
+
+app.use((err, req, res, next) => {
+  if (err.status) {
+    res.status(err.status).send({ msg: err.msg });
+  }
+});
+
+app.all("*", (req, res, next) => {
+  res.status(404).send({ msg: "Not Found" });
+});
```

Comment:
A point relating to the layout - we'd want all of our error handlers to be below our matching endpoints. Not doing this might lead to inconsistencies in how your sever responds, so I'd definitely recommend flipping the order of these two